### DISCUSSION TITLE
preview and themes is not working!

### DISCUSSION THREAD

**Discussion by rennam-martins** [JulianPrieber/littlelink-custom#98] [Issues]
I'm using version 2.7.3 and the preview and themes don't work at all. I downloaded several themes and none worked correctly
![Screenshot_20220708-030756](https://user-images.githubusercontent.com/50037200/177927828-d2860b88-6296-49eb-bbdd-2f37617fe8dc.png)
![Screenshot_20220708-030805](https://user-images.githubusercontent.com/50037200/177927832-a4b7c57f-4c73-4f9d-bc38-439f0c36eb37.png)

**Comment by JulianPrieber** [comment]
Please make sure PHP can write to your LittleLink Custom folder and the web group (for example www-data) owns your folder.

In your installation directory, please check the "themes" folder and see if and what files are in the individual themes folders.

If none of this fixes your issue, kindly provide the following information:

Operating system: `Ubuntu 22.04 LTS...`
Server type: `NGINX/Apache2 ...`
PHP version: `8.1...`

Please double-check if your server meets the requirements on https://llc.bio/req
